### ISSUE TITLE
pip package not working

### ISSUE THREAD

**Issue by yilin-lu** [AguaClara/aide_document#18]
- the package required to upload our own package, twine, constantly throws an "Command not found" error. 
- for some reason, aide_document package cannot be imported even though the version number and license are updated to the correct version

**Comment by ethan92429** [comment]
Please see what I wrote [here](https://github.com/AguaClara/aide_design#updating-the-production-version-v001---v010). Don't use twing. When something says"command no found" it means that probably the command is not installed... ie, the program twine is not installed. You need to upload the package using `python setup.py sdist bdist_wininst upload` as I specified in the ReadMe.

**Comment by ethan92429** [comment]
If you want to use twine, you can, just make sure to install it! https://pypi.python.org/pypi/twine

**Comment by oliver-leung** [comment]
All of us were sure that twine was installed, and uninstalled/reinstalled it numerous times, but it still wouldn't run the command. I tried to manually add the command as well, but I couldn't find it on my computer.
I tried running the `python setup.py sdist bdist_wininst upload` command after changing the version number, but I keep getting a `LookupError: unknown encoding: mbcs` in the Python 3.6 distribution itself. The problem seems to be stemming from the entire `setup.py` file, and I can't localize it (no matter what lines I comment out, it still throws the same error.
I'm thinking that we should just give up on the `pip install` for now? Or would you be able to try and distribute it from your machine?

**Comment by ethan92429** [comment]
First thing to test is see if you are able to distribute anything from your machine. Go ahead and download the [funniest package from this tutorial](http://python-packaging.readthedocs.io/en/latest/minimal.html) and try to upload it to your own pip account. I can look at it if you are not able to get that running.

**Comment by oliver-leung** [comment]
I went through the tutorial, and it seems to work exactly as specified - you can try downloading [this package](https://pypi.org/project/pip_pack_test/) and then running
```
$ python
>>> import pip_pack_test
>>> pip_pack_test.joke()
```
which should output some joke in German. However, when I try going through the upload process (as detailed in the tutorial) on the `aide_document` package, it runs into an authentication error. I've tried copy/pasting the password, messing with the `.pypirc` file, and typing out the password manually, but the error persists at the `upload` step.
Could you try running `python setup.py register sdist upload` in our repo and seeing if that causes problems too?

**Comment by yilin-lu** [comment]
> Upload failed (410): Gone (This API has been deprecated and removed from legacy PyPI in favor of using the APIs available in the new PyPI.org implementation of PyPI (located at https://pypi.org/). For more information about migrating your use of this API to PyPI.org, please see https://packaging.python.org/guides/migrating-to-pypi-org/#uploading. For more information about the sunsetting of this API, please see https://mail.python.org/pipermail/distutils-sig/2017-June/030766.html)
> error: Upload failed (410): Gone (This API has been deprecated and removed from legacy PyPI in favor of using the APIs available in the new PyPI.org implementation of PyPI (located at https://pypi.org/). For more information about migrating your use of this API to PyPI.org, please see https://packaging.python.org/guides/migrating-to-pypi-org/#uploading. For more information about the sunsetting of this API, please see https://mail.python.org/pipermail/distutils-sig/2017-June/030766.html)

This happened after running `python setup.py register sdist upload` on an OS X

**Comment by oliver-leung** [comment]
Okay - I finally got it to work! Turns out that there was some extraneous stuff in my `setup.py` file that was preventing `python setup.py sdist` from packaging the actual code that we needed into the `.tar.gz` installation file. I deleted everything that I could, made sure that I had `packages=find_packages()` within `setup.py`, and then it built everything correctly. Then, I ran `twine upload dist/*`, uploaded onto pypi.org, and now our import statements work.

Man, that was a pain in the ass. I'll write up a CONTRIBUTING.md to go through the upload process and start using tags in this repo to keep our versions in check.
